pkg/c25519/Makefile

```diff
@@ -0,0 +1,32 @@
+PKG_NAME    = c25519
+PKG_URL     = https://www.dlbeer.co.nz/downloads
+PKG_VERSION = 2017-10-05
+PKG_EXT     = zip
+PKG_LICENSE = PD
+
+PKG_BUILDDIR?=$(PKGDIRBASE)/$(PKG_NAME)
+
+ifneq ($(RIOTBASE),)
+include $(RIOTBASE)/Makefile.base
+endif
+
+.PHONY: all clean distclean
+
+prepare: $(PKG_BUILDDIR)/
+
+all: $(PKG_BUILDDIR)/
+	"$(MAKE)" -C $(PKG_BUILDDIR)/src -f $(CURDIR)/Makefile.c25519
+
+$(PKG_BUILDDIR)/: $(PKGDIRBASE)/$(PKG_NAME)-$(PKG_VERSION).$(PKG_EXT)
+	$(Q)$(UNZIP_HERE) -D -d $(PKGDIRBASE) $<
+
+$(PKGDIRBASE)/$(PKG_NAME)-$(PKG_VERSION).$(PKG_EXT):
+	mkdir -p $(PKGDIRBASE)
+	$(Q)$(DOWNLOAD_TO_FILE) $@ $(PKG_URL)/$(PKG_NAME)-$(PKG_VERSION).$(PKG_EXT)
```

Comment:
As this is a crypto library, IMO we must check a hash of the downloaded file here...

Reply:
Sure, do you agree that the current sha512sum is:
```
dbfb4285837ab2ea3d99c448b22877cc7a139ccbaebb1de367e2bec1fd562fe629b389d86603915448078b8fd7e631c8fc9a7d126eb889a1ba0c17611369b190  c25519-2017-10-05.zip
```

Reply:
confirmed

Reply:
Done, please let me know if the current construction is good enough.